src/components/widgets/AppTopNav.tsx

```diff
@@ -211,6 +211,11 @@ const AppTopNav: FunctionComponent<AppTopNavProps> = ({
     route => route.name !== 'APP STORE' && route.name !== 'SIGN IN' && (!route.toggle || toggle[route.toggle!] === true)
   )
 
+  const redirectToLoginPage = () => {
+    sessionStorage.setItem('location', `${location.pathname}${location.search}`)
+    window.location.replace(`${window.location.origin}/sign-in`)
+  }
```

Comment:
Once we allow sign-in with username/password, users may not have a Synapse account. Redirect user to the Open Bridge sign-in page, rather than directly to OneSage.